Druid/DRCogitCodeGenerator.class.st

```diff
@@ -477,6 +477,17 @@ DRCogitCodeGenerator >> visitBranchIfTrue: aDRBranchIfTrue [
 		target: aDRBranchIfTrue falseBranch
 ]
 
+{ #category : #visiting }
+DRCogitCodeGenerator >> visitClosureActivation: aDRClosureActivation [
+
+	self threeOperandCogitRTL: #Add instruction: aDRClosureActivation.
+
+	self
+		oneOperandCogitRTL: #Jump
+		operands: aDRClosureActivation operands
+		instruction: aDRClosureActivation
```

Comment:
I wonder if we don't need in Druid the implementation of a tail call. This idea of activating a closure with a jump (thus reusing the frame) is just a tail call.